### PR TITLE
[EDU-5137] Adjust debugging documentation

### PR DESCRIPTION
**Pull request by guiafonso-ol**
### Changes

Fixes and updates the Debugging with Edge Functions page to match the patterns of a guide, show updated steps and configurations, and provide links to the related reference pages.

### CODE REVIEW

**Review by GabrielAzion — APPROVED** [review]

File: src/content/docs/en/pages/devtools/azion-edge-runtime/debugging/debugging.mdx [hunk 1]
```diff
@@ -25,9 +25,21 @@ The debugging process in **Edge Functions** has the purpose of promoting better
 
 ## How it works
 
-To debug codes, you must create a new function that will use the **log** method. This function prints to a local file a message that was previously sent by a parameter to the function.
+To debug codes, you must create a new [Edge Function](/en/documentation/products/build/edge-application/edge-functions/) that will use the **log** method. This method prints to the console a message that was previously sent by a parameter to the function.
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/azion-edge-runtime/debugging/debugging.mdx:28]:
```suggestion
To debug codes, you must create a new [edge function](/en/documentation/products/build/edge-application/edge-functions/) that will use the **log** method. This method prints to the console a message previously sent by a parameter to the function.
```

Comment (MarianaAguilera) [reply src/content/docs/en/pages/devtools/azion-edge-runtime/debugging/debugging.mdx:28]: product vs. instance

File: src/content/docs/en/pages/devtools/azion-edge-runtime/debugging/debugging.mdx [hunk 2]
```diff
@@ -25,9 +25,21 @@ The debugging process in **Edge Functions** has the purpose of promoting better
...
 
-Check the use of the `console.log` function below:
+### Creating an Edge Function
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/azion-edge-runtime/debugging/debugging.mdx:30]:
```suggestion
### Creating an edge function
```

File: src/content/docs/en/pages/devtools/azion-edge-runtime/debugging/debugging.mdx [hunk 3]
```diff
@@ -42,59 +54,86 @@ Check the use of the `console.log` function below:
...
 
-The message is then collected and made available to be accessed on [Data Stream](/en/documentation/products/observe/data-stream/).
+6. Click the **Save** button to finish the process.
+
+Now that you've created the edge function, you must configure your **Edge Application** to run the function before the logs are accessible.
 
 ---
 
-## Setting up
+### Instantiating the Edge Function
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/azion-edge-runtime/debugging/debugging.mdx:63]:
```suggestion
### Instantiating the edge function
```

File: src/content/docs/en/pages/devtools/azion-edge-runtime/debugging/debugging.mdx [hunk 4]
```diff
@@ -42,59 +54,86 @@ Check the use of the `console.log` function below:
...
+
+---
+
+### Setting up a Data Stream
+
+1. On the **Products Menu**, go to **Observe** > **Data Stream**.
+2. Select an existing stream or create a new one by clicking the **+ Stream** button.
+3. In the *Name** field, give your stream a unique and easy-to-remember name.
+4. In the **Data Settings** section, select **Edge Functions** as the source.
+5. Select the **Edge Functions Event Collector** template. In the **Data Set** box, you will see the preset of variables:
```
Comment (MarianaAguilera) [inline src/content/docs/en/pages/devtools/azion-edge-runtime/debugging/debugging.mdx:95]:
```suggestion
5. Select the **Edge Functions Event Collector** template. In the **Data Set** box, you'll see the preset of variables:
```

File: src/content/docs/pt-br/pages/devtools/azion-edge-runtime/debugging/debugging.mdx [hunk 5]
```diff
@@ -25,9 +25,21 @@ O processo de debugging em **Edge Functions** tem por finalidade promover maior
 
 ## Como funciona
 
-Para depuração de códigos, você deve criar uma nova função que irá utilizar o método **log**. Essa função escreve para um arquivo local a mensagem passada previamente por parâmetro a ela.
+Para depurar códigos, você deve criar uma nova [Edge Function](/pt-br/documentacao/produtos/build/edge-application/edge-functions/) que utilizará o método **log**. Esse métodoo imprime no console uma mensagem que foi previamente enviada como parâmetro para a função.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/azion-edge-runtime/debugging/debugging.mdx:28]:
```suggestion
Para depurar códigos, você deve criar uma nova [edge function](/pt-br/documentacao/produtos/build/edge-application/edge-functions/) que utilizará o método **log**. Esse métodoo imprime no console uma mensagem que foi previamente enviada como parâmetro para a função.
```

File: src/content/docs/pt-br/pages/devtools/azion-edge-runtime/debugging/debugging.mdx [hunk 6]
```diff
@@ -25,9 +25,21 @@ O processo de debugging em **Edge Functions** tem por finalidade promover maior
...
 
-Verifique a aplicação da função `console.log` abaixo:
+### Criar uma Edge Function
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/azion-edge-runtime/debugging/debugging.mdx:30]:
```suggestion
### Criar uma edge function
```

File: src/content/docs/pt-br/pages/devtools/azion-edge-runtime/debugging/debugging.mdx [hunk 7]
```diff
@@ -40,62 +52,83 @@ Verifique a aplicação da função `console.log` abaixo:
        addEventListener("fetch", (event) => {
            event.respondWith(handleRequest(event.request));
        });
-    }
 ```
 
-A mensagem, então, é coletada e fica disponível para ser acessada via [Data Stream](/pt-br/documentacao/produtos/observe/data-stream/).
+6. Clique no botão **Salvar** para finalizar o processo.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/azion-edge-runtime/debugging/debugging.mdx:57]:
```suggestion
6. Clique no botão **Save** para finalizar o processo.
```

File: src/content/docs/pt-br/pages/devtools/azion-edge-runtime/debugging/debugging.mdx [hunk 8]
```diff
@@ -40,62 +52,83 @@ Verifique a aplicação da função `console.log` abaixo:
...
+
+Agora que você criou a edge function, deve configurar sua **Edge Application** para executar a função para que os logs sejam acessíveis.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/azion-edge-runtime/debugging/debugging.mdx:59]:
```suggestion
Agora que você criou a edge function, deve configurar sua **edge application** para executar a função para que os logs sejam acessíveis.
```

File: src/content/docs/pt-br/pages/devtools/azion-edge-runtime/debugging/debugging.mdx [hunk 9]
```diff
@@ -40,62 +52,83 @@ Verifique a aplicação da função `console.log` abaixo:
...
+6. Escolha a instância da sua edge function.
+7. Clique no botão **Save**.
 
-**Data Source**: é a plataforma Azion que gerará os eventos de onde você deseja coletar dados. Selecione uma opção.
+Agora a função será executada de acordo com a regra e seus logs estarão prontos para serem capturados pelo **Data Stream**.
 
-**Template**: definição das variáveis. Selecione **Custom Template** para criar uma nova configuração de dados e inclua as seguintes variáveis no campo **Data Set**:
+---
+
+### Configurar um Data Stream
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/devtools/azion-edge-runtime/debugging/debugging.mdx:89]:
```suggestion
### Configure um data stream
```

**Review by MarianaAguilera — APPROVED** [review]

**Pull request merged**